dts/bindings/pinctrl/nxp,s32ze-pinctrl.yaml

```diff
@@ -58,12 +58,11 @@ description: |
     - open drain disabled
     - slew rate 4 (see description in property below).
 
-  Additionally, Safe Mode is always disabled (reset value) and configuration that
-  only applies to LVDS pads, which are not supported, default to reset values:
-    - termination resistor disabled
-    - receiver single ended
-    - current reference control disabled
-    - Rx current boost disabled.
+  Additionally, Safe Mode is always disabled (reset value) and the following configurations
+  that only apply to LVDS pads are already supported, defaulting to their reset values:
+    - termination resistor
```

Comment:
what are reset values? why don't we just keep like before: (i.e `termination resistor disabled`)
one more thing that needs explain is that why you remove the `receiver single ended`? it is not LVDS pad?

Reply:
- I updated the description for more clear
- receiver single ended belongs to RCVR field is not specific for LVDS, so I won't support in this PR, will support later if needed

Reply:
The intention of this paragraph was to document the default values for the settings that are not exposed through dt properties, e.g. safe mode and receiver single ended. Termination resistor, current reference, rx current boost -> you already documented them on each prop.